comm/src/main/scala/coop/rchain/comm/transport/GrpcTransportReceiver.scala

```diff
@@ -64,10 +57,10 @@ object GrpcTransportReceiver {
           // TODO cancel queues when peer is lost
```

Comment:
Buffers were on _io_ scheduler which also doesn't make sense because the are not doing any _io_.